docs/default.nix

```
@@ -209,17 +209,37 @@ let
 
                 maintainersText =
                   if maintainers == [ ] then
-                    "This module has no [dedicated maintainers](../../modules.md#maintainers)."
+                    "This module has no [dedicated maintainers](../../modules.md#maintainers).\n"
```

If we keep this rendering format, it would be better to keep components standalone and add the empty line in the main part in lines 219--223 (old):

```suggestion
                    "This module has no [dedicated maintainers](../../modules.md#maintainers)."
```